shared-module/displayio/__init__.c

```diff
@@ -190,7 +190,7 @@ void reset_displays(void) {
         if (((uint32_t) fourwire->bus) < ((uint32_t) &displays) ||
             ((uint32_t) fourwire->bus) > ((uint32_t) &displays + CIRCUITPY_DISPLAY_LIMIT)) {
             busio_spi_obj_t* original_spi = fourwire->bus;
-            if (original_spi == board_spi()) {
+            if (original_spi == common_hal_board_get_spi()) {
```

Comment:
Could you put in a comment here about the fact the `original_spi` or `common_hal_board_get_spi()` might be null, and if so why, and how that's handled? I'm having trouble figuring out the different possible cases.